packages/plugin-ngrok/README.md

```diff
@@ -0,0 +1,43 @@
+This package contains a Shopify CLI plugin that enables the creation of ngrok tunnels from `shopify app dev`, 
```

Comment:
I'd create a `plugins` directory inside [the docs](https://github.com/Shopify/cli/tree/main/docs) directory, move this one there, and [add a new section to the README](https://github.com/Shopify/cli/blob/main/docs/README.md) with a reference to it.